hugo/layouts/partials/authors.html

```diff
@@ -0,0 +1,67 @@
+{{ $authorsPages := .Site.GetPage "/authors" }}
+{{ $authorsPages := $authorsPages.Resources.ByType "page" }}
+{{ $articleAuthors := .Params.Authors | default false }}
+
+<ul class="authors">
```

Comment:
this should not a ul because ul can only contain list items 
or the dtails element should be wrapped inside an li , or don't use ul here